### DISCUSSION TITLE
Cross-Platform Support (Linux, Mac & etc.)

### DISCUSSION THREAD

**Discussion by kisoqual** [baffler/Transparent-Twitch-Chat-Overlay#201] [Ideas]
when these builds come and have you interest on these platforms? 
let me know which version and platform will support

**Comment by kisoqual** [comment]
and also its possible to build this? let me know

**Comment by baffler** [comment]
I'm probably going switch the project to use WebView2 soon. You can see the roadmap here: https://learn.microsoft.com/en-us/microsoft-edge/webview2/roadmap

It seems they do have plans on having macOS and Linux support. So it may be possible to have this working for those platforms.

**Comment by Greenteally** [comment]
And here i am waiting for linux support
